### PR TITLE
Update EIP-211: Typos fix

### PR DESCRIPTION
**Pull request by MaxweLL22-22**
Fix: Corrected typos in source files

**Changes**

Corrected typo in /EIPS/eip-2982.md (Line 309)

"Tung" → "Tongue"

Corrected typo in /EIPS/eip-7748.md (Line 117)

"implementator" → "implementer"

Corrected typo in /EIPS/eip-7748.md (Line 255)

"ommitted" → "omitted"

Corrected typo in /EIPS/eip-2780.md (Line 58)

"appearant" → "apparent"

Corrected typo in /EIPS/eip-225.md (Line 413)

"readded" → "re-added, read"

Corrected typo in /EIPS/eip-211.md (Line 23)

"implementors" → "implementers"

Corrected typo in /EIPS/eip-3102.md (Line 32)

"implementors" → "implementers"

Corrected typo in /EIPS/eip-7898.md (Line 19)

"infact" → "in fact"

Corrected typo in /EIPS/eip-7898.md (Line 45)

"augument" → "argument, augment"

Corrected typo in /EIPS/eip-7898.md (Line 47)

"annouce" → "announce"

**Purpose**

Improved code accuracy and professionalism by fixing typos.

### CODE REVIEW

File: EIPS/eip-225.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ tests := []struct {
   }, {
     // Ensure that pending votes don't survive authorization status changes. This
     // corner case can only appear if a signer is quickly added, removed and then
-    // readded (or the inverse), while one of the original voters dropped. If a
+    // re-added, read (or the inverse), while one of the original voters dropped. If a
```
Comment (g11tech) [inline EIPS/eip-225.md:413]: this isn't comprehensible anymore

**Comment by SamWilsn** [comment]
I'm _very_ quickly going through typo fix PRs. Unfortunately this PR has a conflict, so I'm going to close it, but feel free to create a new one if the issue still exists.
